docs/dev-guide/contributing/branching.rst

```diff
@@ -78,7 +74,7 @@ Hotfix
 When a hotfix needs to be made, a branch will be created from the most recent
 ``x.y.z`` release tag. The fix will be made (via pull request from a personal
 fork to the hotfix branch), a new tag will be built from the tip of the hotfix
-branch, and the hotfix branch can be merged to ``x.y-dev``.
+branch, and the fix can be cherry-picked forward to ``x.y-release`` and ``master``.
```

Comment:
I was thinking that hotfixes could just be made against the `x.y-release` branch and then cherry picked back to master.

Reply:
Cool, updated.